src/relay_pic/main.c

```diff
@@ -0,0 +1,162 @@
+/* 
+ * File:   pinConfigMain.c
+ * Author: zacha
+ *
+ * Created on March 1, 2020, 3:21 PM
+ */
+

```

Comment:
Nitpick but we usually remove these generated lines. Usually we have multiple people editing the file anyway and git keeps track of authorship.